test/services/licences/fetch-agreements.service.test.js

```diff
@@ -30,24 +30,42 @@ describe('Fetch Agreements service', () => {
         code: 'S127'
       })
 
-      await LicenceAgreementHelper.add({
+      licenceAgreementData = {
         endDate: new Date('2040-05-01'),
         financialAgreementId: financialAgreement.id,
         licenceRef,
         startDate: new Date('2022-04-01'),
         signedOn: new Date('2022-04-01')
+      }
+    })
+
+    describe('and the agreement has not been deleted', () => {
+      beforeEach(async () => {
+        await LicenceAgreementHelper.add(licenceAgreementData)
+      })
+
+      it('returns the matching agreements data', async () => {
+        const results = await FetchAgreementsService.go(licenceRef)
+
+        expect(results[0]).to.equal({
+          startDate: new Date('2022-04-01'),
+          signedOn: new Date('2022-04-01'),
+          endDate: new Date('2040-05-01'),
+          financialAgreement: { id: '970168ce-06c3-4823-b84d-9da30b742bb8', code: 'S127' }
+        }, { skip: ['id'] })
       })
     })
 
-    it('returns the matching agreements data', async () => {
-      const results = await FetchAgreementsService.go(licenceRef)
+    describe('and the agreement has been deleted', () => {
+      beforeEach(async () => {
+        await LicenceAgreementHelper.add(licenceAgreementData.deletedAt = new Date('2024-07-31 13:06:27.666'))
```

Comment:
I wouldn't bother hard coding a date here as it doesn't really matter what the date is, just that it's not null.
```suggestion
        await LicenceAgreementHelper.add(licenceAgreementData.deletedAt = new Date())
```